cmd/go-semantic-release/commands/changelog.go

```diff
@@ -8,7 +8,9 @@ import (
 
 func init() {
 	changelogCmd.Flags().Bool("checks", false, "Check for missing values and envs")
+	changelogCmd.Flags().Bool("overwrite", false, "Overwrite the content of the changelog. Default is to prepend the new changelog to the existing file.")
```

Comment:
Should we add the flags to the release commands and write the changelog there too?

Reply:
This would only improve the workflow if the users running the release command on their local machine right? 
So If yes, I think they can run two commands :D

Reply:
True, but there are custom setups. But this can be done in a extra PR